core/print_api.php

```diff
@@ -2125,6 +2125,17 @@ function get_filesize_info( $p_size, $p_unit ) {
 	return sprintf( lang_get( 'max_file_size_info' ), number_format( $p_size ), $p_unit );
 }
 
+/**
+ * Returns target tag to be added in attachment links
+ * @return string
+ */
+function get_attachment_link_target() {
```

Comment:
By convention, we usually prefix functions with the API's name, so this should be called *print_attachment_link_target()*